app/pages/Accounts/AccountDetailsPage/AccountDetailsPage.tsx

```diff
@@ -76,7 +76,11 @@ export default withAccountSync(function DetailsPage() {
                         component={accountChanged ? ToAccounts : BuildSchedule}
                     />
                     <Route path={routes.ACCOUNTS_ADDRESS}>
-                        <ShowAccountAddress account={account} asCard />
+                        <ShowAccountAddress
```

Comment:
You have only added the verification here, but not in the `TransactionsAndAddress` view.

Reply:
Yes, but that was deliberately, due to concerns regarding vertical space, 
but i can add it to the TransactionsAndAddress view, if you think it should be there

Reply:
Get some input from Jens on how to design it. It should be an additional step to get your account address without verifying it at the same time, if we want to help users protect themselves.